snowflake_blueprints/upload_file.py

```diff
@@ -145,15 +320,15 @@ def main():
             database=args.database,
             schema=args.schema,
             warehouse=args.warehouse
-        ))
+        )).execution_options(autocommit=True)
```

Comment:
What's the reason for adding this and the change? 

This could lead to undesired results where dropping a table succeeds, but inserting the table fails. 
Is this purely for performance? 
I would document either way.

Reply:
I need to retest this. I think I intially had to add it in because the TRUNCATE table function would never actually work if autocommit=True wasn't on. Since I'm using DROP instance of TRUNCATE now, this might no longer be necessary.

Reply:
Appears I can remove this without issues. Thanks for the catch.